core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileClassWarehouse.scala

```diff
@@ -96,9 +96,7 @@ case class SQLCleanAndAlignIdsProfileResult(
   override def convertToSeq: Seq[String] = {
     Seq(appIndex.toString, sqlID.toString)
   }
-  override def convertToCSVSeq: Seq[String] = {
-    Seq(appIndex.toString, sqlID.toString)
-  }
+  override def convertToCSVSeq: Seq[String] = convertToSeq
```

Comment:
FYI: I see some refactoring calling convertToSeq implicitly from `convertToCSVSeq`. While this is valid for some classes, it should be only limited to classes that have numeric fields (no string fields).
There is a difference between `convertToSeq` and `convertToCSVSeq`.
The latter wraps fields of type String with `\"` so that it will skip special characters.
`convertToSeq` is meant to be used by the textWriter because it does not add this extra quotes.
In summary, if fields are of type strings then they should be protected by double quote first within the convertToCSVSeq.

Finally, It could cause bugs because another contributer assumes that he is calling the right method `convertToCSVSeq` while it is not.

Reply:
That's ok to keep them. I wanted just to give a heads-up about how doing such refactor might introduce a bug.